### PR TITLE
Fix build break caused by merge

### PR DESCRIPTION
**Pull request by agocke**
@dotnet/roslyn This should fix the build break in master

### CODE REVIEW

File: src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs [hunk 1]
```diff
@@ -1765,7 +1765,7 @@ public class C
 
         [Fact, Trait(Traits.Feature, Traits.Features.TypeInferenceService)]
         [WorkItem(6765, "https://github.com/dotnet/roslyn/issues/6765")]
-        public void TestDefaultStatement1()
+        public async void TestDefaultStatement1()
```
Comment (jaredpar) [inline src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs]: async void ahhhhh!!!!!

File: src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs [hunk 2]
```diff
@@ -1775,12 +1775,12 @@ static void Main(string[] args)
...
     }
 }";
-            Test(text, "global::System.ConsoleModifiers", testNode:false);
+            await TestAsync(text, "global::System.ConsoleModifiers", testNode:false);
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.TypeInferenceService)]
         [WorkItem(6765, "https://github.com/dotnet/roslyn/issues/6765")]
-        public void TestDefaultStatement2()
+        public async void TestDefaultStatement2()
```
Comment (jaredpar) [inline src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs]: async void ahhhhhh!!!!

Comment (Pilchie) [inline src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs]: It's actually *safe* in unit tests, but I'd still rather see it updated.

**Comment by Pilchie** [comment]
Thanks @agocke - forgot that tests should have been re-run after @CyrusNajmabadi's PR went in.

File: src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs (hunk 1 above)
Comment (agocke) [inline src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs]: Doesn't XUnit hook this so the exception is propagated?

Comment (agocke) [inline src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs]: Anyway, regardless, best practices -- changing.

Comment (Pilchie) [inline src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs]: Yes, Xunit handles it, but let's still put tasks there.

**Comment by davkean** [comment]
:+1:

**Comment by VSadov** [comment]
LGTM

**Comment by jaredpar** [comment]
If this is just fixing a build break should we merge now?  The builds have finished, it's just running tests at the moment.

**Comment by agocke** [comment]
Yes, merging

**Comment by amcasey** [comment]
@jaredpar, strictly speaking the new tests (i.e the ones that didn't build) could fail.  But yes, we should probably merge.

**Comment by agocke** [comment]
(I did run those tests locally)

**Pull request merged**